test/e2e/network/service.go

```diff
@@ -1334,7 +1339,8 @@ var _ = SIGDescribe("Services", func() {
 		svcName := fmt.Sprintf("%v.%v.svc.%v", serviceName, f.Namespace.Name, framework.TestContext.ClusterDNSDomain)
 		ginkgo.By("Waiting for endpoints of Service with DNS name " + svcName)
 
-		execPodName := e2epod.CreateExecPodOrFail(f.ClientSet, f.Namespace.Name, "execpod-", nil)
+		execPod := e2epod.CreateExecPodOrFail(f.ClientSet, f.Namespace.Name, "execpod-", nil)
+		execPodName := execPod.Name
 		cmd := fmt.Sprintf("wget -qO- http://%s:%d/", svcName, port)
```

Comment:
Use curl

Reply:
Missed to address in new commit. Once all jobs completes, I would push another commit replacing all possible `wget` with `curl` from these files.

Reply:
Done.